lib/template.html

```diff
@@ -110,8 +112,8 @@
 {% endif %}
 
 {% if page.url %}
```

Comment:
Is there a case when we don't wan't to add `canonical` or `og:url` ? If not 👍  we can get rid of this condition in a separate PR.